src/stores/SpaceStore.tsx

```diff
@@ -111,6 +116,18 @@ export class SpaceStoreClass extends AsyncStoreWithClient<IState> {
         this.emit(UPDATE_SELECTED_SPACE, this.activeSpace);
         this.emit(SUGGESTED_ROOMS, this._suggestedRooms = []);
 
+        // view last selected room from space
+        const spaceId = space?.roomId || LAST_VIEWED_ROOMS_HOME;
+        const lastViewedRooms = JSON.parse(window.localStorage.getItem(LAST_VIEWED_ROOMS));
+        const roomId = lastViewedRooms[spaceId];
+
+        if (roomId) {
+            dis.dispatch({
+                action: "view_room",
+                room_id: roomId,
+            });
+        } // TODO: Handle else
```

Comment:
Is there any desired behaviour for this?

Reply:
in the case of no prior room I think it should go to space home, or for the home space just to the first room in the room list maybe